ethsigner/signer/multiplatform/src/test/java/tech/pegasys/ethsigner/signer/multiplatform/FileBasedSigningMetadataFileTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.multiplatform;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static tech.pegasys.ethsigner.signer.multiplatform.MetadataFileFixture.KEY_FILE;
+import static tech.pegasys.ethsigner.signer.multiplatform.MetadataFileFixture.NO_PREFIX_LOWERCASE_ADDRESS;
+import static tech.pegasys.ethsigner.signer.multiplatform.MetadataFileFixture.PASSWORD_FILE;
+import static tech.pegasys.ethsigner.signer.multiplatform.MetadataFileFixture.PREFIX_ADDRESS;
+import static tech.pegasys.ethsigner.signer.multiplatform.MetadataFileFixture.load;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+
+class FileBasedSigningMetadataFileTest {
+
+  @Test
+  void matchingMetadataFileWithoutPrefixShouldHaveExpectedName() {
+    final FileBasedSigningMetadataFile fileBasedSigningMetadataFile =
+        load(NO_PREFIX_LOWERCASE_ADDRESS, KEY_FILE, PASSWORD_FILE);
+
+    assertThat(fileBasedSigningMetadataFile.getFilename()).matches(NO_PREFIX_LOWERCASE_ADDRESS);
```

Comment:
feels like the "FileBasedSIgningMetaDataFile" doesn't do much - other than strip "toml" from the file ... do we need to do such indepth testing? tbh - the "stripping" function could be a static function, and then just test it directly (then again, its pretty simple, so only requires minimal testing)

Reply:
You can ditch one of these tests - FileBasedSIgningMetaDataFile doesn't deal with prefixes - so no need to prove it.
You can also probably not worry about the toml extension - do we care what they name them? why gatekeep?

Reply:
Finally - you can probably get away without using resources for any of this - you can make up fake paths and send them in - then you don't need to use the TestFixture for this class at all